packages/data-explorer-ui/src/components/Filter/components/Filters/filters.tsx

```diff
@@ -84,17 +85,20 @@ export const Filters = ({
   onFilter,
 }: FiltersProps): JSX.Element => {
   return (
-    <FilterList disabled={disabled}>
-      {categoryViews.map((categoryView) => (
-        <Filter
-          content={renderFilterMenu(categoryView, onFilter)}
-          key={categoryView.key}
-          tags={renderFilterTags(categoryView, onFilter)}
-          Target={(props): JSX.Element =>
-            renderFilterTarget(categoryView, props)
-          }
-        />
-      ))}
-    </FilterList>
+    <>
+      <SearchAllFilters categoryViews={categoryViews} onFilter={onFilter} />
```

Comment:
For flexibility of components, I think we should move the render of `SearchAllFilters` to https://github.com/clevercanary/data-explorer/blob/5376ef153faab249c6da4dd3e055b926729ad40c/packages/data-explorer-ui/src/views/ExploreView/exploreView.tsx#L109 (above the render of the `Filters` component)